artemis/reporting/severity.py

```diff
@@ -17,6 +17,7 @@ class Severity(str, Enum):
     ReportType("forti_vuln"): Severity.HIGH,
     ReportType("globalprotect_vuln"): Severity.HIGH,
     ReportType("lfi_vulnerability"): Severity.HIGH,
+    ReportType("rce_vulnerability"): Severity.HIGH,
```

Comment:
Let's add an end-to-end test where you integrate the Karton with the Reporter and check resulting HTML. You can find examples in test/reporting/